plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -305,7 +312,8 @@ def addSshKeyToUser(self, user, sshkey):
             try:
                 user.keys.create({
                     'title': sshkey['name'],
-                    'key': sshkey['file']})
+                    'key': sshkey['file'],
+                    'expires_at': sshkey['expires_at']})
```

Comment:
Is this option present with all API versions we support, or is it better to only pass it when not `None`?

Reply:
It's better to only pass it when it's not `None` for sure, also from what I can see in https://docs.gitlab.com/archives/ the oldest doc available is `10.3` version of gitlab which is only 18 months old, so I guess we don't have an option here.

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/2450/commits/77e3fba79d0fb535cf2444b2a6c896a5d3ebddd1.